recipe/activate.bat

```diff
@@ -126,6 +126,14 @@ IF "%USE_NEW_CMAKE_GEN_SYNTAX%" == "1" (
 
 pushd %VSINSTALLDIR%
 CALL "VC\Auxiliary\Build\vcvars%BITS%.bat" -vcvars_ver=@{vcvars_ver} %WindowsSDKVer%
+:: if this didn't work and CONDA_BUILD is not set, we're outside
+:: conda-forge CI so retry without vcvars_ver, which is going to
+:: fail on local installs that don't match our exact versions
+if %ERRORLEVEL% neq 0 exit 1 (
+  if "%CONDA_BUILD%" == "" (
+    CALL "VC\Auxiliary\Build\vcvars%BITS%.bat"
+  )
+)
```

Comment:
I'm not 100% sure if we want to keep %WindowsSDKVer% here (based on what we can find with `:GetWin10SdkDir`); according to the [docs](https://learn.microsoft.com/en-us/cpp/build/building-on-the-command-line?view=msvc-170#vcvarsall-syntax), the argument is optional.